lte/gateway/c/core/oai/lib/store/sqlite.cpp

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2022 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <sqlite3.h>
+#include <cmath>
+#include <vector>
+#include "lte/protos/subscriberdb.pb.h"
+#include "lte/gateway/c/core/oai/lib/store/sqlite.hpp"
```

Comment:
Move this include to the top, as per the [Google Style guide](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)